Dev script is broken

```
Keegan-Pebble-56:clay keegan$ npm run dev

> pebble-clay@1.0.1 dev /Users/keegan/workspace/clay
> gulp dev

(node:31062) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[22:15:51] Using gulpfile ~/workspace/clay/gulpfile.js
[22:15:51] Starting 'clean-js'...
[22:15:51] Starting 'clean-sass'...
[22:15:51] Finished 'clean-sass' after 4.97 ms
[22:15:51] Starting 'sass'...
[22:15:51] Finished 'sass' after 11 ms
[22:15:51] Finished 'clean-js' after 29 ms
[22:15:51] Starting 'js'...
[22:15:51] Finished 'js' after 444 ms
[22:15:51] Starting 'dev-js'...
events.js:160
throw er; // Unhandled 'error' event
^

Error: Cannot find module 'message_keys' from '/Users/keegan/workspace/clay'
at /Users/keegan/workspace/clay/node_modules/resolve/lib/async.js:46:17
at process (/Users/keegan/workspace/clay/node_modules/resolve/lib/async.js:173:43)
at ondir (/Users/keegan/workspace/clay/node_modules/resolve/lib/async.js:188:17)
at load (/Users/keegan/workspace/clay/node_modules/resolve/lib/async.js:69:43)
at onex (/Users/keegan/workspace/clay/node_modules/resolve/lib/async.js:92:31)
at /Users/keegan/workspace/clay/node_modules/resolve/lib/async.js:22:47
at FSReqWrap.oncomplete (fs.js:117:15)
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -109,6 +109,7 @@ gulp.task('dev-js', ['js', 'sass'], function() {
     .transform('deamdify')
     .transform(sassify, sassifyOptions)
     .transform(autoprefixify, autoprefixerOptions)
+    .ignore('message_keys')
     .bundle()
     .pipe(source('dev.js'))
     .pipe(gulp.dest('./tmp/'));

